hyperv/tools/hypestv/src/windows/mod.rs

```diff
@@ -275,8 +295,11 @@ pub async fn main(driver: DefaultDriver) -> anyhow::Result<()> {
                                 .handle_command(cmd)
                                 .await?;
                         }
+                        InteractiveCommand::Quit => {
```

Comment:
lol, I'm surprised this wasn't here in v0. how were you quitting before this? like, via ctrl-d?

Reply:
Yeah, ctrl-d.